source/installation/deploy-minio-standalone.rst

```diff
@@ -40,6 +40,17 @@ deployments are best suited for evaluation and initial development environments.
    For deployments that *require* using network-attached storage, use
    NFSv4 for best results.
 
+Pre-Existing Data
+~~~~~~~~~~~~~~~~~
+
+When starting a new MinIO server, the storage devices may have existing data.
+MinIO adds this data as buckets and objects in the MinIO deployment.
+
+Once you start the MinIO server, all interactions with the data must be done through the S3 API.
```

Comment:
So this isn't true for standalone single-drive because we're just a layer on top of the FS. Users can change files directly on the backend.

Reply:
Removed